LoRaEngine/test/LoRaWanNetworkServer.Test/LoRaPayloadDecoderTest.cs

```diff
@@ -156,6 +176,28 @@ public async Task When_Decoder_Is_DecoderValueSensor_Return_In_Value(string deco
             Assert.Equal(expectedValue, decodedPayloadValue.Value);
         }
 
+        [Theory]
+        [InlineData("DecoderHexSensor", "1234", "31323334", 1)]
```

Comment:
Confusing to me to use the fport, as it is ignored all the way in the implementation. I would rather use a fixed fport.

Reply:
Should I change this in both the tests for DecoderValueSensor and DecoderHexSensor, @fbeltrao ?

Reply:
Maybe it would be interesting to run using different fports to ensure that it returns always the same value.

The included decoders don't use it, but the caller expects it to be there, so we can't remove it from the method signature.

Reply:
That's how it's now. The same calls are made using different fports for both DecoderValueSensor and DecoderHexSensor.